README.md

```diff
@@ -269,19 +289,13 @@ export const lookId = 1
 
 ### Step 3 - Build and run the demo
 
-#### Node server
```

Comment:
@slopp I am working on helping a customer, and I am using Node for my server (and I am successfully running it so far). Is there any reason we are deleting the instruction for the Node server here?

Reply:
@Ianlooker the node server is still available and the npm commands will still work, however I removed the demo documentation around the node server because the capability relies on a webpack dev server which is really intended for frontend testing - the npm webpack commands do not provide a demo backend that is realistic for customers. The example had confused some of our prospects who thought no backend would be necessary.

Reply:
@slopp I see, thanks for explaining. I think we should bring back the documentation for node.js example because 
1/ Many customers are still using node.js demo (I am working on a case with a customer and they only want to build a prototype with front end, they wouldn't need backend)
2/ Examples for internal enablement for customer engineering and support team are using node.js 

Since the README.md in the root folder is quite long, I am thinking we could make 2 separate md file for node and python (with disclaimer what node.js is for front-end demo and python is for full stack). Would that work for you?

Reply:
@ianlooker would start a new PR since this one is merged and closed?

Reply:
Hi @slopp, thanks for clarifying. I appreciate your help and improvement to the python example 🙏🏻  I'll play around with it. For the node: I'll bring it back in a new PR, I am checking here with you to make sure I don't miss anything, and it seems that we both reach a consent here. I'll tag you in that new PR if you would like to take a look. Thank you